extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -403,7 +403,9 @@ describe('amp-a4a', () => {
         const doc = fixture.doc;
         const a4aElement = doc.createElement('amp-a4a');
         const a4a = new AmpA4A(a4aElement);
-        return a4a.maybeRenderAmpAd_(null).then(rendered => {
+        a4a.adUrl_ = 'http://foo.com';
+        a4a.maybeRenderAmpAd_ = function() { return Promise.resolve(false); }
```

Comment:
Yeah...  I think that's my mistake in writing the docs.  :-)  The output from `validateAdResponse_()` _can_ be null:

```javascript
return verifySignature(
          new Uint8Array(creative), signature, publicKeyInfos).then(isValid => {
            return isValid ? creative : null;
          });
```

And I think the cut-out for checking `creative && ...` is knee-jerk more than anything.

The intended behavior is: "If the thing validates, then it should pass a valid creative `ArrayBuffer` along.  Anything else (i.e., `null`) should be interpreted as invalid."  But I lost track of who was responsible for handling the `null` case.  By adding the cut-out, I prevented `null` from entering `maybeRenderAmpAd()` in the pipeline flow.  But then this test forced `null` in.  Since `maybeRenderAmpAd` doesn't have a check for that, it errors when trying to convert `null` to utf8.

I guess formerly that error was correctly interpreted as render failure, but now it's not?